.tekton/scripts/check-partner-tasks.sh

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd "$SCRIPTDIR/../.." || exit 1
+
+# Expected partner task directory structure:
+# task/partners/<task name>/<version>/<task file>
+
+if [ ! -e "partners/" ]; then
+    exit 0
+fi
+
+check_dir_structure() {
+    issue_found=0
+    for task_dir in partners/*; do
+        task_name=$(basename "$task_dir")
+        for version_dir in "$task_dir"/*; do
+            task_version=$(basename "$version_dir")
+            if ! echo "$task_version" | grep "^[0-9.]\+$" >/dev/null 2>&1; then
```

Comment:
If someone by mistake name a version directory as `1..2` (with double . in the middle), they will not get this error right?

Reply:
Do we need to consider `semantic versioning` regex here?

Reply:
The task directory structure was added as a catalog by referencing the tekton catalog. Version is not specific to a semantic version scheme, and so far I also don't see any requirement of this "complicated" versioning methodology. But, I still modified the regex to validate the version that should be in format N.M and updated the error message to avoid ambiguity.

Reply:
Okay .. got it